Signal/src/ViewControllers/SendExternalFileViewController.m

```diff
@@ -64,6 +64,7 @@ - (void)threadWasSelected:(TSThread *)thread
         return;
     }
 
+    [ThreadUtil addThreadToProfileWhitelistIfEmptyContactThread:thread];
```

Comment:
It's not great that we have to remember to call `addThreadToProfileWhitelistIfEmptyContactThread` anywhere that we might be sending a message to a thread that we've just created, but I was a bit wary of pulling this isn't ThreadUtil or MessageSender, not least because we only want to do this when the user sends a "normal" message - a text message or attachment - not any other message type.  We could probably identify "normal" messages... tough call.